core.html

```diff
@@ -169,6 +169,8 @@ <h2>Recognizing and handling peer DIDs</h2>
         capture group 1, <code>transform</code> in capture group 2, and <code>encnumbasis</code> in capture group 3.
     </p>
 
+    <p>Note that peer DIDs can also be used in DID URIs, with query parameters. Currently, the only query parameter formally defined for peer DIDs is <code>initial-state</code>, discussed <a target="#initial-state">below</a>. However, other parameters may be used by a given implementation, so recipients of a peer DID URI should at least be prepared to parse the DID into its base DID value and its query string portion (which begins with the <code>?</code> character. All query parameter values in peer DID URIs are base64url encoded.</p>
```

Comment:
Are we planning to just use the default version-time stuff from did-core which is why we have this statement: "Currently, the only query parameter formally defined for peer DIDs is <code>initial-state</code>,"

Reply:
Ooh, good catch.I will remove the "only query parameter" language.

I think peer dids need to support version stamping. I don't know about version timing; each party sees evidence for an update accumulate in different orders and at different times. @kdenhartog or @peacekeeper , can you point me to docs on the versioning parameters?

Reply:
Fixed.

Reply:
The only documentation I'm aware of is here: https://w3c.github.io/did-core/#generic-did-url-parameters